dataflow-migrate-schedules/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.2.1.RELEASE</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>io.spring</groupId>
+	<artifactId>migrateschedule</artifactId>
+	<version>1.0.0.BUILD-SNAPSHOT</version>
+	<name>Schedule Migrator</name>
+	<description>Migrates SCDF Schedules to the 2.3 format</description>
+
+	<properties>
+		<java.version>1.8</java.version>
+		<spring-cloud-data-flow.version>2.3.0.RC1</spring-cloud-data-flow.version>
+		<deployer.version>2.1.0.RELEASE</deployer.version>
+	</properties>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-task-dependencies</artifactId>
+				<version>2.2.0.BUILD-SNAPSHOT</version>
```

Comment:
I guess this can now be changed to use `2.2.1.RELEASE`?

Reply:
Removing task dependency and `@EnableTask` it isn't providing us any real support for this effort, it is batch providing the services.

Reply:
After talking with Michael.  Task will remain in that it provides a task execution track for our users.   Updating to 2.2.1.RELEASE